crt_portal/cts_forms/templates/forms/report.html

```diff
@@ -3,27 +3,61 @@
 {% block head %}
 {{ wizard.form.media }}
 {% endblock %}
-
 {% block content %}
+  <span class="usa-sr-only">
```

Comment:
I am thinking if we create the text in the view and pass these in to the template, it will make internationalization easier.

Here is some background on how to do that in Django: https://docs.djangoproject.com/en/2.2/topics/i18n/translation/

Translation is not in the thin slice but, I like to make it easier on us or others when that happens. Happy work on it with you if you want; What do you think?

Reply:
Thanks for the note! I'll keep this in mind going forward. And I would love to work on this together during an upcoming pairing session. Since translation is not in the thin slice of our current scope, I would be comfortable shipping this current PR without translation support; we can add translation support for this feature in a future PR.

Reply:
Sounds good, to be clear, we would not translate it now either way, I was only suggesting to add the logic to the view and pass the variables to the template. But it sounds like we are on the same page with it not being necessary for this PR.

Reply:
Yep!